cypress_test/plugins/index.js

```diff
@@ -101,6 +101,11 @@ function pickTests(filename, foundTests) {
 			testsToRun = [];
 		}
 	}
+
+	if (!process.env.UPDATE_SCREENSHOT) {
```

Comment:
Can we improve the condition a bit - it is hard to read: if (! something) { remove_something(); } ; would be preferred to have if (something) { add_something(); } - if possible...

Reply:
In this case, It is not possible